packages/core/src/components/cv-data-table/cv-data-table.vue

```diff
@@ -34,7 +34,7 @@
           </div>
         </div>
 
-        <div v-if="$slots.actions && !batchActive" class="bx--toolbar-content">
+        <div v-if="($slots.actions && !batchActive) || $listeners.search" class="bx--toolbar-content">
```

Comment:
Search should not be available if batchActive === true

Perhaps 

(($slots.actions || $listeners.serach) && !batchActive)

Reply:
Fixed.